jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -270,6 +270,7 @@ def test_parse_jwst_nircam_level2(self, imviz_helper):
         phot_plugin.counts_factor = (data.meta['photometry']['conversion_megajanskys'] /
                                      data.meta['exposure']['exposure_time'])
         assert_allclose(phot_plugin.counts_factor, 0.0036385915646798953)
+        assert_quantity_allclose(phot_plugin.flux_scaling, 3.631e-3, rtol=1e-3)
```

Comment:
Why is the rtol so relaxed? Also, `flux_scaling` is not a Quantity.
```suggestion
        assert_allclose(phot_plugin.flux_scaling, 3.631e-3)
```

I am also surprised that `'aperture_sum_mag'` value in this same test did not change. It should have.